CNV-58838: Added <shortdesc> to Managing VMs sections (1) (#102888)
openshift/openshift-docs: modules/virt-storage-wizard-fields-web.adoc

[id="virt-storage-wizard-fields-web_{context}"]
= Storage fields

[role="_abstract"]

> To configure the source and characteristics of a virtual machine's disk, use the Storage fields in the manifest or web console.

Reviewer: The abstract should explain why a user wants to do this - what's the use case of configuring these? What benefit does it provide?

> Configure the source and characteristics of a virtual machine's disk by using the Storage fields in the manifest or web console.

Reviewer 2: ```suggestion
To configure the source and characteristics of a virtual machine's disk, use the Storage fields in the manifest or web console.
```
